client/api/ApiWrapper.tsx

```diff
@@ -0,0 +1,152 @@
+import {PublicUserData, User} from "@types"
+import { Value } from "react-native-reanimated";
+
+const API_URL = 'http://localhost:3000/';
+
```

Comment:
Maybe make this a typescript class? That way it can have variables because you need to be able to store the JWT token. Another thing that would be nice to store is the `userId`  of the user logged in. Because that way, you can just do `getUser()` and it will return the currently logged in user. Essentially the wrapper should make it as easy as possible to get data from the API when coding the UI.

The UI can create an instance of this class, and then get whatever they need easily. For example, they can do

```
api = new ApiWrapper()
api.signIn(user,pass) // only need to call this once
api.getUser() //this returns the currently logged in user
...
api.logout() // this one just clears the JWT 
api.isLoggedIn() // this one should check if they are logged in (if id and jwt stored)
```

you should have a signIn method that stores the JWT, and needs to be called once when it starts. Eventually we will have to store the JWT on the user's phone that way they can stay signed in even after they close the app. But for now, just store it as a variable after they call `signIn`